video/cloud-client/analyze/video_detect_logo.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+
+# [START video_detect_logo]
+
+from google.cloud import videointelligence
+from google.cloud.videointelligence import enums
```

Comment:
Delete

Reply:
Delete what? The import statements? Should I move them above the region tag?

Or just delete the enums line since your solution for features is cleaner?

Reply:
Just enums